scheduler/plugin/host.go

```diff
@@ -217,6 +224,14 @@ func k8sApiPodFn(ctx context.Context, mod wazeroapi.Module, stack []uint64) {
 	stack[0] = uint64(marshalIfUnderLimit(mod.Memory(), pod, buf, bufLimit))
 }
 
+func k8sApiPodInfoFn(ctx context.Context, mod wazeroapi.Module, stack []uint64) {
```

Comment:
Completely same as `k8sApiPodFn`?

Reply:
I guess L231 is supposed to be `.podInfo`

Reply:
Oh, I will fix this:)